.github/workflows/update-valkey-hashes.yml

```
@@ -0,0 +1,78 @@
+name: Commit new hash into valkey-hashes
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: The version of Valkey to create.
+        type: string
+        required: true
+    secrets:
+      PAT_TOKEN:
+        description: PAT token for valkey-hashes repo.
+        required: true
+
+jobs:
+  commit-hash:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate inputs
+        run: |
+          if [[ ! "${{ inputs.version }}" =~ ^([0-9]+\.[0-9]+\.[0-9]+|unstable)$ ]]; then
+            echo "Invalid version format '${{ inputs.version }}'. Expected format: x.y.z or 'unstable'."
+            exit 1
+          fi
+
+      - uses: actions/checkout@v4
```

```suggestion
      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```
Can you also change the checkout to actual hashes
https://github.com/valkey-io/valkey/blob/1a975faf79c35cbbd6c846b118b3060af4ccf559/.github/workflows/ci.yml#L16C9-L16C81